src/cc/tools/qfsfileenum_main.cc

```diff
@@ -61,7 +63,8 @@ main(int argc, char **argv)
                 port = atoi(optarg);
                 break;
             case 'f':
-                filename = optarg;
+                filename         = optarg;
```

Comment:
Again, here with the indentation like this, lines that aren't actually a part of this commit are touched... that unnecessarily changes git last touch history and assigns the hash to a change that was entirely irrelevant.